multistream_test.go

```diff
@@ -677,7 +677,7 @@ func (rob *readonlyBuffer) Close() error {
 	return nil
 }
 
-func TestNegotiatThenWriteFail(t *testing.T) {
+func TestNegotiateThenWriteFail(t *testing.T) {
```

Comment:
My understanding is that changes to go test names are fairly harmless even though they're public functions...